android/src/main/java/com/klaviyoreactnativesdk/KlaviyoReactNativeSdkModule.kt

```diff
@@ -14,135 +15,136 @@ import com.klaviyo.analytics.model.ProfileKey
 import java.io.Serializable
 import kotlin.reflect.KVisibility
 
-class KlaviyoReactNativeSdkModule internal constructor(private val context: ReactApplicationContext) :
```

Comment:
I flattened this class structure. We had this so that the base class could be different depending on old/new arch by using different source sets depending on architecture (also deleted the old/new source sets). We can assess and re-add what is needed when we add TM support